parts/linux/cloud-init/artifacts/ci-syslog.service

```diff
@@ -0,0 +1,9 @@
+[Unit]
```

Comment:
you need to add all of the files in VHD to https://github.com/Azure/AgentBaker/blob/master/vhdbuilder/packer/vhd-image-builder-gen2.json for each sku you care about (likely all of them, except windows)